.gitignore

```diff
@@ -10,3 +10,7 @@ drafts
 .DS_Store
 .sass-cache
 .ruby-gemset
+public/css/funky_world_cup.css
+public/css/funky_world_cup.css.map
+public/css/funky_world_cup_light.css
```

Comment:
Can you also remove the `assets/sass/funky_world_cup_light.scss` file as well? I'm pretty sure we're not using it, I should have left it there by mistake when implementing the whole template structure.

Reply:
Good catch! I removed it. I guess we don't need to ignore them anymore, but I'll them here just in case.